Examples/Python/python/acts/examples/odd_light.py

```diff
@@ -0,0 +1,321 @@
+    sstripNecPositionsZ = [-2950, -2550, -2200, -1850, -1550, -1300]
+    sstripNecHalfThickness = 15
+    sstripNecBinning = [
+        ProtoBinning(Binning.r, Binning.bound, 230, 710, 3, 1),
+        detector.phiBinning(40, 1),
+    ]
+    sstripBarrelPositionsR = [260, 360, 500, 660]
+    sstripBarrelHalfThickness = 25
+    sstripBarrelZbinning = ProtoBinning(Binning.z, Binning.bound, -1100, 1100, 21, 1)
+    sstripBarrelPhiBinnings = [[40, 1], [56, 1], [78, 1], [102, 1]]
+    sstripBarrelBinning = [
+        [sstripBarrelZbinning, detector.phiBinning(phiBinning[0], phiBinning[1])]
+        for phiBinning in sstripBarrelPhiBinnings
+    ]
+
+    sstrip = necBarrelPec(
+        "ODD_sstrip",
+        sstripRrange,
+        sstripZdivisions,
+        sensitivesKdt,
+        sstripNecPositionsZ,
+        sstripNecHalfThickness,
+        sstripNecBinning,
+        sstripBarrelPositionsR,
+        sstripBarrelHalfThickness,
+        sstripBarrelBinning,
+        llevel,
+    )
+
+    # Long strip section ##########################################################
+
+    lstripRrange = [sstripRrange[1], detRrange[1]]
+    lstripZdivisions = [detZrange[0], -1250, 1250, detZrange[1]]
+
+    lstripNecPositionsZ = [-3000, -2600, -2250, -1900, -1600, -1300]
+    lstripNecHalfThickness = 15
+    lstripNecBinning = [
+        ProtoBinning(Binning.r, Binning.bound, 720, 1020, 2, 1),
+        detector.phiBinning(40, 1),
+    ]
+    lstripBarrelPositionsR = [830, 1030]
+    lstripBarrelHalfThickness = 35
+    lstripBarrelZbinning = ProtoBinning(Binning.z, Binning.bound, -1100, 1100, 21, 1)
+    lstripBarrelPhiBinnings = [[40, 1], [56, 1]]
+    lstripBarrelBinning = [
+        [lstripBarrelZbinning, detector.phiBinning(phiBinning[0], phiBinning[1])]
+        for phiBinning in lstripBarrelPhiBinnings
+    ]
+
+    lstrip = necBarrelPec(
+        "ODD_lstrip",
+        lstripRrange,
+        lstripZdivisions,
+        sensitivesKdt,
+        lstripNecPositionsZ,
+        lstripNecHalfThickness,
+        lstripNecBinning,
+        lstripBarrelPositionsR,
+        lstripBarrelHalfThickness,
+        lstripBarrelBinning,
+        llevel,
+    )
+
+    # Container of beam pipe and pixel
+    det = detector.CylindricalDetectorContainer(
+        name="ODD",
+        extent=None,
+        volumes=[bp, pix, pst, sstrip, lstrip],
+        layers=None,
+        binning=Binning.r,
+        loglevel=llevel,
+    )
+
+    # Pixel Endcap
+    detConfig = DetectorBuilder.Config()
+    detConfig.name = "ODD"
+    detConfig.builder = det.builder()
+    detConfig.auxiliary = "Detector[" + detConfig.name + "]"
+
+    detBuilder = DetectorBuilder(detConfig, detConfig.auxiliary, logging.VERBOSE)
+    return detBuilder.construct(geoContext)
+
+
+def main():
+    # Parse the command line arguments
+    p = argparse.ArgumentParser()
+    p.add_argument(
+        "-i",
+        "--input",
+        type=str,
+        default="odd-light.gdml",
+        help="GDML input file (optional)",
```

Comment:
The gdml file is optional?

Reply:
Copy and past error, although the odd_light will build from other inputs in the future, for the moment optional is not correct, removed it.